xcodeproj/internal/xcodeproj.bzl

```diff
@@ -267,6 +290,9 @@ def make_xcodeproj_rule(*, transition = None):
             allow_empty = False,
             aspects = [xcodeproj_aspect],
         ),
+        "_allowlist_function_transition": attr.label(
+            default = "@bazel_tools//tools/allowlists/function_transition_allowlist",
+        ),
```

Comment:
I don't see this being used anywhere. Is this needed in this PR?

Reply:
Yes. It enables the use of the transition that is set on `cfg`.